docs/example-api-calls/google-workspace.md

```diff
@@ -48,20 +48,22 @@ node tools/psoxy-test/cli.js -u [your_psoxy_url]/psoxy-gcal/calendar/v3/calendar
 ## Directory
 
 ### Domains
+Replace `[customer_id]` with your Google Workspace account ID, see: https://developers.google.com/admin-sdk/directory/reference/rest/v1/domains/get
```

Comment:
are you *certain* `my_customer` doesn't work? I think it should ...

Reply:
reverting that change; it seems to work but I couldn't find any refs in their docs where that alias/wildcard is defined